<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OpenMPT Plugin Settings</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>374</width>
     <height>167</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Interpolation:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QComboBox" name="interpolate_combo"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Stereo separation:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QSlider" name="stereo_separation">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSpinBox" name="stereo_separation_spin">
      <property name="maximum">
       <number>200</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QCheckBox" name="use_filename">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Use filename as song title</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="defaults_button">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Restore Defaults</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="2">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stereo_separation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>stereo_separation_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stereo_separation_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>stereo_separation</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>defaults_button</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>restoreDefaults()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>restoreDefaults()</slot>
 </slots>
</ui>
